en/TOC.md

```diff
@@ -6,76 +6,81 @@
   - [What's New in v1.2](whats-new-in-v1.2.md)
 - [Get Started](get-started.md)
 - Deploy
-  - Deploy TiDB Cluster
-    - [On AWS EKS](deploy-on-aws-eks.md)
-    - [On GCP GKE](deploy-on-gcp-gke.md)
-    - [On Azure AKS](deploy-on-azure-aks.md)
-    - [On Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
-    - In Self-managed Kubernetes
-      - [Prerequisites](prerequisites.md)
-      - [Configure Storage Class](configure-storage-class.md)
-      - [Deploy TiDB Operator](deploy-tidb-operator.md)
-      - [Configure TiDB Cluster](configure-a-tidb-cluster.md)
-      - [Deploy TiDB Cluster](deploy-on-general-kubernetes.md)
-      - [Initialize TiDB Cluster](initialize-a-cluster.md)
-      - [Access TiDB Cluster](access-tidb.md)
-    - [Deploy TiDB Cluster on ARM64 Machines](deploy-cluster-on-arm64.md)
-  - [Deploy a TiDB Cluster across Multiple Kubernetes Clusters](deploy-tidb-cluster-across-multiple-kubernetes.md)
-  - [Deploy Heterogeneous Cluster](deploy-heterogeneous-tidb-cluster.md)
-  - [Deploy TiFlash](deploy-tiflash.md)
-  - [Deploy DM](deploy-tidb-dm.md)
+  - On-Premises
+    - [Prerequisites](prerequisites.md)
+    - [Configure Storage Class](configure-storage-class.md)
+    - [Deploy TiDB Operator](deploy-tidb-operator.md)
+    - [Configure TiDB Cluster](configure-a-tidb-cluster.md)
+    - [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md)
+    - [Deploy TiDB on ARM64 Machines](deploy-cluster-on-arm64.md)
```

Comment:
Public cloud vendors also have arm machines.

Reply:
> Public cloud vendors also have arm machines.

Got it. I moved it to a parallel position with On-premises and Public Cloud K8s in https://github.com/pingcap/docs-tidb-operator/pull/1478/commits/de15a1c60b388c4f6346a4036bcfadac97d33aca.